aspnetcore/blazor/components.md

```diff
@@ -67,7 +67,8 @@ To render a component from a page or view, use the `RenderComponentAsync<TCompon
 
 ```cshtml
 <div id="Counter">
-    @(await Html.RenderComponentAsync<Counter>(new { IncrementAmount = 10 }))
+    @(await Html.RenderComponentAsync<Counter>(RenderMode.Static, 
```

Comment:
You should only use `RenderMode.Static` for content that isn't interactive. Since the `Counter` component is intended to be interactive, I think you should use `RenderMode.ServerPrerendered` instead. However, you we don't support passing in parameters for that mode yet, so you'll have to forgo showing that here.